### PR TITLE
Multi-user task manual decisions #267

### PR DESCRIPTION
**Pull request by Lengocduong**

### CODE REVIEW

File: docs/_locale/de/LC_MESSAGES/action-types.po [hunk 1]
```diff
@@ -173,6 +173,24 @@ msgstr "In diesem Beispiel enthält das Formular der Mehrbenutzer-Aufgabe eine e
...
+msgstr "Die Mehrbenutzer-Aufgabe unterstützt auch manuelle Entscheidungen. Wenn nach einer Mehrbenutzer-Aufgabe ein exklusives Gateway folgt, so können Sie die Buttons konfigurieren, welche die möglichen Entscheidungen an diesem Gateway widerspiegeln. Die Buttons werden im Formular der Mehrbenutzer-Aufgabe angezeigt."
+
+msgid "Configuring decision options for a multi-user task"
+msgstr "Entscheidungsoption für eine Mehrbenutzer-Aufgabe konfigurieren"
+
+msgid "Select the decision type 'Manual' for the exclusive gateway. Then choose the decision option from the dropdown menu:"
+msgstr "Wählen Sie den Entscheidungstyp „Manuell“ für das Gateway aus. Die Entscheidungsmöglichkeit wird anschließend über das Dropdown-Menü selektiert:"
+
+msgid "*All users have chosen this option*"
+msgstr "Alle Nutzer haben diese Option ausgewählt."
```
Comment (SilentGert) [inline docs/_locale/de/LC_MESSAGES/action-types.po]: What about the markdown highlighting here: `*...*`

File: docs/include_action-types/multi-user-task.rst [hunk 2]
```diff
@@ -44,3 +44,19 @@ The mapping to *Approvals* creates a variable with the same type as the *Approve
 Completing the tasks that the multi-user task generated adds the *Approved* values (Yes or No) to the *Approvals* list.
 
 To use the results of a multiple approval, use a :ref:`javascript-action` to implement whatever logic consolidates the list of results in the list into a single decision.
+
+Manual decision (Exclusive gateway)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The multi-user task supports manual decisions.
+If an exclusive gateway follows a multi-user task, you can configure the buttons, which reflect the possible decisions to this gateway.
```
Comment (TimKam) [inline docs/include_action-types/multi-user-task.rst]: *which* -> *that*

**Review by TimKam — CHANGES_REQUESTED** [review]
One detail, plus Gert's comment. Otherwise, 👍

File: docs/include_action-types/multi-user-task.rst [hunk 3]
```diff
@@ -44,3 +44,19 @@ The mapping to *Approvals* creates a variable with the same type as the *Approve
...
+The buttons are displayed in the form of the multi-user task.
+
+.. figure:: /_static/images/action-types/multi-user-task/manual-decision.png
+
+   Configuring decision options for a multi-user task
+
+Select the decision type 'Manual' for the exclusive gateway. Then choose the decision option from the dropdown menu:
```
Comment (TimKam) [inline docs/include_action-types/multi-user-task.rst]: I forgot: *dropdown* -> *drop-down*

**Review by TimKam — APPROVED** [review]

**Pull request merged**